README.md

```diff
@@ -1,4 +1,134 @@
-# Trampoline V2
+# docker-ci-helper
 
-This is a shell script for running another script in a Docker
-container.
+This repository contains `trampoline_v2.sh`, a shell script for
+running another script in a Docker container. Primary use case of this
+shell script is to use it as a test driver on various CI systems. By
+utilizing Docker, your tests will be very portable so that you can run
+them on any CI systems as well as on your workstations with minimum
+efforts.
+
+## trampoline_v2.sh
```

Comment:
After reading the whole script top to bottom, I'm still having a little difficulty understanding its purpose.

Could you add a section discussing this script's inputs and outputs, at a very general level?

Reply:
I updated the first section of the README. WDYT?

Reply:
Nice.  That new paragraph forms a clear image in my brain.